admin/client/stores/CurrentListStore.js

```diff
@@ -17,50 +18,59 @@ var active = {
 
 var page = defaultPage();
 
-function defaultPage () {
+function defaultPage() {
 	return {
 		size: 100,
 		index: 1
 	};
 }
 
+var _rowAlert = defaultRowAlert();
+
+function defaultRowAlert() {
+	return _rowAlert = {
+		success: false,
+		fail: false
+	};
+}
+
 var CurrentListStore = new Store({
 	getList () {
 		return _list;
 	},
-	getAvailableColumns () {
+	getAvailableColumns() {
```

Comment:
Mind undoing these whitespace changes please? as per above, trying to get consistency.